.circleci/config.yml

```diff
@@ -45,6 +45,7 @@ jobs:
           command: sudo iptables -I INPUT -m conntrack --ctstate INVALID -j DROP
       - run:
           name: Stand up sandbox
+          no_output_timeout: 30m
```

Comment:
why are we increasing this?

Reply:
```suggestion
```

Reply:
Key registration takes a long time, so I needed to update this.

Reply:
can we put some echo or print instead of extending this?

Reply:
No, it's not part of this repository.